lib/ansible/modules/cloud/amazon/ec2_vol_facts.py

```diff
@@ -59,57 +60,65 @@
 import traceback
 
 try:
-    import boto.ec2
-    from boto.exception import BotoServerError
-    HAS_BOTO = True
+    from botocore.exceptions import ClientError
```

Comment:
This module should handle BotoCoreError and ClientError. ProfileNotFound should be handled by boto3_conn and doesn't need to be imported by the module.